### PR TITLE
Remaining obligation message

### PR DESCRIPTION
**Pull request by jonsterling**

### CODE REVIEW

File: src/redprl/metalanguage/evaluate.fun [hunk 1]
```diff
@@ -134,7 +134,7 @@ struct
 
          val check =
            if subgoalsCount = 0 then () else
-             RedPrlLog.print RedPrlLog.WARN (pos, Fpp.hsep [Fpp.text @@ Int.toString subgoalsCount, Fpp.text "Remaining Obligations"])
+             RedPrlLog.print RedPrlLog.WARN (pos, Fpp.hsep [Fpp.text @@ Int.toString subgoalsCount, Fpp.text @@ "Remaining Obligation" ^ (if subgoalsCount = 1 then "" else "s")])
```
Comment (favonia) [inline src/redprl/metalanguage/evaluate.fun]: I suggest using the whole words "Obligation" and "Obligations" so that future localization would be easier.

**Comment by cangiuli** [comment]
It might be nice for the "obligations remain in file" message to be located at the end of the file.

**Comment by jonsterling** [comment]
@cangiuli Thank you for the review! I will resolve this by adding source locations to the new metalanguage.

**Comment by favonia** [comment]
What is blocking this PR now? I think this is worth merging.

**Comment by jonsterling** [comment]
@favonia If I have a moment in the next couple days, I'll get it up to date.

**Pull request closed without merging**